src/workers/cdc/primary.ts

```diff
@@ -622,6 +622,29 @@ const onPostChange = async (
     if (isChanged(data.payload.before!, data.payload.after!, 'title')) {
       await clearPostTranslations(con, data.payload.after!.id, 'title');
     }
+
+    const postMetrics: (keyof ChangeObject<Post>)[] = [
+      'upvotes',
+      'downvotes',
+      'comments',
+      'awards',
+    ];
+
+    if (
+      postMetrics.some((field) =>
+        isChanged(data.payload.before!, data.payload.after!, field),
```

Comment:
Should we make `isChanged` be able to accept field as array as well, and embed `some` in `isChanged` when field is an array?

Reply:
good idea